matRadGUI.m

```diff
@@ -2451,7 +2451,8 @@ function getPlnFromGUI(handles)
 
 try
     cst = evalin('base','cst');
-    if sum(strcmp('TARGET',cst(:,3))) > 0 && get(handles.checkIsoCenter,'Value')
+    if (sum(strcmp('TARGET',cst(:,3))) > 0 && get(handles.checkIsoCenter,'Value')) || ...
+            (sum(strcmp('TARGET',cst(:,3))) > 0 && ~isfield(pln,'isoCenter'))
        pln.isoCenter = ones(pln.numOfBeams,1) * matRad_getIsoCenter(cst,ct);
```

Comment:
...s there a reason for not just calling getPlnFromGUI for all edits in GUI (regarding plan) and using pln variable if it exists?...

not that I am aware of. @becker89 do you know something?!

I think we should try to get this fixed asap so we can submit the revision of the matRad paper to Med Phys!